R/find_exports.R

```diff
@@ -18,7 +18,7 @@ find_exports <- function(clean_lns) {
 
 # Finds lines which contain #[extendr] (allowing additional spaces)
 find_extendr_attrs_ids <- function(lns) {
-  which(stringi::stri_detect_regex(lns, "#\\s*\\[\\s*extendr\\s*\\]"))
+  which(stringi::stri_detect_regex(lns, "#\\s*\\[\\s*extendr(\\s*\\(.+\\))?\\s*\\]"))
```

Comment:
I find this regex difficult to read. Can we use a raw string? I believe we're requiring R >= 4.0 already anyways.

Reply:
Thanks, let's use a raw string.